api/v1beta1/lokistack_types.go

```diff
@@ -338,6 +511,13 @@ type LokiStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced",displayName="Node Placement"
 	Template *LokiTemplateSpec `json:"template,omitempty"`
+
+	// Tenants defines the lokistack-gateway component tenants configuration spec.
```

Comment:
wording `defines the per-tenant authentication and authorization spec for the lokistack-gateway component`